core/tests/protractor_desktop/coreEditorAndPlayerFeatures.js

```diff
@@ -25,14 +25,164 @@ var users = require('../protractor_utils/users.js');
 var workflow = require('../protractor_utils/workflow.js');
 
 
-var AdminPage = require('../protractor_utils/AdminPage.js');
 var CreatorDashboardPage =
   require('../protractor_utils/CreatorDashboardPage.js');
 var ExplorationEditorPage =
   require('../protractor_utils/ExplorationEditorPage.js');
 var ExplorationPlayerPage =
   require('../protractor_utils/ExplorationPlayerPage.js');
 var LibraryPage = require('../protractor_utils/LibraryPage.js');
+var ExplorationEditorPage =
+  require('../protractor_utils/ExplorationEditorPage.js');
+
+describe('Enable correctness feedback and set correctness', function() {
+  var explorationEditorPage = null;
+  var explorationEditorMainTab = null;
+  var explorationEditorSettingsTab = null;
+  var explorationPlayerPage = null;
+  var currentExplorationIndex = 0;
+  var EXPLORATION_TITLE = 'Dummy Exploration';
+  var explorationTitle = null;
+  var libraryPage = null;
+  var correctOptions = [
+    ['MultipleChoiceInput', 'Correct!'],
+    ['TextInput', 'One'],
+    ['NumericInput', 3]
+  ];
+
+  var enableMarkCorrectness = function() {
```

Comment:
Maybe a better name would be enableCorrectnessFeedbackSetting.

Reply:
Updated, thanks!